pootle/apps/pootle_project/models.py

```diff
@@ -396,6 +395,12 @@ def clean(self):
                 _('"%s" cannot be used as a project code', self.code)
             )
 
+    def directory_exists(self):
+        """Checks if the actual directory for the translation project
+        exists on-disk.
```

Comment:
This is checking if the directory for the project exists, not for the TP.

Nit: Use "on disk"